src/components/select/editForm/Select.edit.data.js

```diff
@@ -143,26 +144,29 @@ export default [
     template: '<span>{{ item.label }}</span>',
     valueProperty: 'key',
     dataSrc: 'url',
+    lazyLoad: false,
     onSetItems(component, form) {
       const newItems = [];
 
       eachComponent(form.components, (component, path) => {
-        newItems.push({
-          label: component.label || component.key,
-          key: path
-        });
+        if (component.input) {
+          newItems.push({
+            label: component.label || component.key,
+            key: `data.${path}`
+          });
+        }
       });
 
       return newItems;
     },
     data: {
-      url: '/form/{{ data.resource }}'
+      url: '/form/{{ data.data.resource }}'
```

Comment:
This doesn't look correct. If the resource is on the wrong path, we should figure out why it is in the wrong place instead of changing the reference to it.